pkg/util/test/util.go

```diff
@@ -1829,6 +1838,197 @@ func ValidateTelemetryInstalled(pxImageList map[string]string, cluster *corev1.S
 	return nil
 }
 
+/* TODO: We need to make this work for spawn
+func createMetrixCollectorDeploymentObject(cluster *corev1.StorageCluster, pxImageList map[string]string) (*appsv1.Deployment, error) {
```

Comment:
I believe we don't need this. Ideally we should be reading a spec from the file and changing the individual components as needed by the test in future. Should we just remove this?

Reply:
Done, removed